### ISSUE TITLE
Upgrade to apollo-server 2.14.x or later

### ISSUE THREAD

**Issue by JohnAlbin** [AmazeeLabs/gatsby-starter-amazee#72]
We are currently stuck on 2.13.1 (the minor version before the latest 2.14.2).

When updating the yarn.lock file to 2.14.2, [our tests start failing](https://travis-ci.com/github/AmazeeLabs/gatsby-starter-amazee/builds/169974352) with:

```
[mock:api] > gatsby-starter-amazee@0.1.0 mock:api /home/travis/build/AmazeeLabs/gatsby-starter-amazee
[mock:api] > ts-node mocks/swapi.ts
[mock:api] 
[mock:api] 
[mock:api] /home/travis/build/AmazeeLabs/gatsby-starter-amazee/node_modules/apollo-server/src/exports.ts:6
[mock:api]   GraphQLUpload,
[mock:api]   ^
[mock:api] TypeError: Cannot redefine property: GraphQLUpload
```

2.14.2 include a security fix, so this is kind of urgent. See https://github.com/AmazeeLabs/gatsby-starter-amazee/pull/68

**Comment by JohnAlbin** [comment]
I'm trying to write a small reproducible test case to report the issue to apollo-server. https://github.com/JohnAlbin/test-apollo-server

**Comment by JohnAlbin** [comment]
PR is at #69

**Comment by JohnAlbin** [comment]
Fixed in #69
